s3/s3i_test.go

```diff
@@ -239,9 +208,10 @@ func (s *ClientTests) TestRegions(c *C) {
 	errs := make(chan error, len(aws.Regions))
 	for _, region := range aws.Regions {
 		go func(r aws.Region) {
-			s := s3.New(s.s3.Auth, r)
-			b := s.Bucket("goamz-" + s.Auth.AccessKey)
-			_, err := b.Get("non-existent")
+			s := s3.New(s.s3.Auth, r, aws.SignV2)
+			b, err := s.Bucket("goamz-" + s.Auth.AccessKey)
+			c.Assert(err, IsNil)
```

Comment:
I'd prefer not to use c.Assert in a loop like this, so that a single failure won't stop the test. How about:
```go
if !c.Check(err, IsNil) {
  errs <- err
}
_, err = b.Get("non-existent")
```